R/package.R

```diff
@@ -145,11 +145,7 @@ get_bootstrap_version <- function(template, config_path = NULL, package = NULL)
 
     cli::cli_abort(
```

Comment:
Can't use `config_abort()` because it's called before the pkg object is created.